core/build.gradle.kts

```diff
@@ -196,20 +200,41 @@ kotlin {
             }
         }
 
-        val jsMain by getting {
+        val jsAndWasmSharedMain by creating {
```

Comment:
I'd use a different name for this source set, e.g. `jsWasmJs` similar to how in stdlib we use `nativeWasm`

Reply:
This name was suggested by reviewers in `kotlinx.coroutines` in sake of readability between shared code between `js` and `wasm` sourceset and `wasmJs` target sourceset. But I can rename it to `jsWasmJs ` if you see this is still better.

Reply:
I think `jsAndWasmSharedMain` is more readable.

Reply:
I have a better idea - `commonJs`. It's a code shared by Js flavors, be it classic Js or Wasm-Js.